main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative './app'
+
```

Comment:
- [ ] I am getting an error when I try to run your code, checkout the screenshot below and kindly fix the issue. Please make sure you app is running according to the project [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md) before you submit for another review.  

![ruby error](https://user-images.githubusercontent.com/65852621/224413660-2ea9f723-4d0f-490e-a6d2-aa6c51624788.PNG)